src/selectors/getPartitionedProjects.js

```diff
@@ -0,0 +1,9 @@
+import {createSelector} from 'reselect';
+
+import partition from 'lodash-es/partition';
+
+import getAllProjects from './getAllProjects';
+
+export default createSelector([getAllProjects], projects =>
+  partition(projects, {isArchived: false}),
```

Comment:
EXTREMELY NITPICKY ALERT i think you can just pass the string `'isArchived'` here rather than `{isArchived: false}`